pkg/environment/environment.go

```diff
@@ -101,3 +98,19 @@ func GetInsprEnvironment() string {
 func GetInsprAppID() string {
 	return getEnv("INSPR_APP_ID")
 }
+
+// GetBrokerWritePort returns environment variable that contains given broker's write port
+func GetBrokerWritePort(broker string) string {
+	return getEnv(fmt.Sprintf("INSPR_SIDECAR_%s_WRITE_PORT", strings.ToUpper(broker)))
+}
+
+// GetBrokerReadPort returns environment variable that contains given broker's read port
+func GetBrokerReadPort(broker string) string {
```

Comment:
I think this wont be used

Reply:
We can remove it later, after the k8s part is finished (just to be sure this really wont be used)